pavelib/database.py

```diff
@@ -63,23 +61,19 @@ def update_local_bokchoy_db_from_s3():
       with all the migrations
     * If not then check if there is a copy up at s3
     * If so then download then extract it
-    * Otherwise apply migrations as usual
+    * Otherwise apply migrations as usual and push the new cache
```

Comment:
nit: This seems to me to be overloading the "update_local_bokchoy_db_from_s3" method with additional functionality. 

E.g. without the "upload" addition, a devstack user could use the task to get the latest, without inadvertently trying to upload (in which they probably don't have the right credentials and will get ... ? dunno, I'm guessing a BuildFailure).

I'm also thinking it may be fine to refactor in the next PR.